src/hooks/useImportingWallet.ts

```diff
@@ -327,6 +329,12 @@ export default function useImportingWallet({ showImportModal = true } = {}) {
                     handleSetImporting(false);
                   }
 
+                  setTimeout(() => {
```

Comment:
curious how this will look like for folks who will get the backup prompt as well below (lines 338-354)